tasks/task524_parsinlu_food_aspect_classification.json

```diff
@@ -11,7 +11,7 @@
     "Input_language": "fa",
     "Output_language": "en",
     "instruction_language": "en",
-    "Definition": "In this task, you will be given a food review and a question about the reviewer's sentiment toward one aspect of the food. You have to answer the question based on the review. Classify the answers into: \"no sentiment expressed\", \"very negative\", \"negative\", \"neutral\", \"positive\",  \"very positive\", and \"mixed\". The mixed category indicates reviews where none of the sentiments are dominant (mix of positive and negative, or borderline cases); hence it is hard to detect the primary sentiment. Also, assign neutral label to reviews that express no clear sentiment toward an entity or any aspect of it. The \"no sentiment expressed\" label should be assigned to the reviews where the given aspect didn't discuss in the text. \"very negative\" and \"very positive\" labels indicate extreme sentiment toward an aspect.",
+    "Definition": "In this task, you will be given a food review and a question about the reviewer's sentiment toward one aspect of the food. You have to infer the answer to the question from the review. Classify the reviewer's sentiment into: \"no sentiment expressed\", \"very negative\", \"negative\", \"neutral\", \"positive\",  \"very positive\", and \"mixed\". The mixed category indicates reviews where none of the sentiments are dominant (mix of positive and negative, or borderline cases); hence it is hard to detect the primary sentiment. Also, assign neutral label to reviews that express no clear sentiment toward an entity or any aspect of it. The \"no sentiment expressed\" label should be assigned to the reviews where the given aspect didn't discuss in the text. \"very negative\" and \"very positive\" labels indicate extreme sentiment toward an aspect.",
```

Comment:
isn't using `... question from the review and Classify ..` better?

Reply:
By combining them, it's going to be a really long sentence. If you think it's better, I can fix it now.

Reply:
yes your right but I still think the definition isn't complete . can we change it into something like `You have to infer the answer to the question from the review and classify it. Classify the reviewer's sentiment into...`

Reply:
Sure! I updated them.